test/osb_and_plugin_test/provision_timeout_test/provision_timeout_test.go

```diff
@@ -0,0 +1,22 @@
+package provision_timeout
+
+import (
+	"github.com/Peripli/service-manager/pkg/web"
+	. "github.com/onsi/ginkgo"
+	"net/http"
+)
+
+var _ = Describe("Provision", func() {
+	Context("when provision takes more than the configured timeout", func() {
```

Comment:
maybe add another test for less than timeout just to see if it works with current test configuration